modules/configuring-rbac.adoc

```diff
@@ -0,0 +1,106 @@
+// Module is included in the following assemblies:
+//
+// * managing_cluster_configuration/managing-openshift-cluster-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-rbac_{context}"]
+= Configuring RBAC
+
+You must configure RBAC to provide the sufficient access to the users to work with the default instance.
+
+The `defaultPolicy` of the instance is an empty string, which means no role is assigned automatically. Though users can log into the instance, they have no permissions to view anything or perform any tasks in the Argo CD UI or CLI.
+
+The instance includes the following two groups:
+
+* `system:cluster-admins`: The group only applies to the temporary `kube-admin` credentials and can be ignored.
+* `cluster-admins`: You can add your user to this group to enable them to perform tasks, such as deploying an application, in the Argo CD web console.
+
+[NOTE]
+====
+_Restrict default permissions:_
+
+Ensure that you always use either an empty string or a deny-all type of role for the `defaultPolicy` parameter because permissions granted in it cannot be revoked. Hence, it is not recommended to set the `defaultPolicy` parameter in a way that grants permissions.
+====
+
+.Prerequisite
+
+* You have login credentials to access the {OCP} cluster with `cluster-admin` privileges.
+* You have installed the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/cli_tools/openshift-cli-oc[`oc` CLI].
```

Comment:
maybe point to /latest than explicitly /4.19 ? but that depends on whether you want  a specific version here or not